saleor/graphql/product/mutations/attributes.py

```diff
@@ -160,15 +171,29 @@ def clean_attribute(cls, instance, cleaned_input):
         if input_slug is None:
             cleaned_input["slug"] = slugify(cleaned_input["name"])
         elif input_slug == "":
-            raise ValidationError({"slug": "The attribute's slug cannot be blank."})
+            raise ValidationError(
+                {
+                    "slug": ValidationError(
+                        "The attribute's slug cannot be blank.",
+                        code=AttributeErrorCode.ATTRIBUTE_SLUG_BLANK,
```

Comment:
This is simply a "value required" type of error.

Reply:
I've changed it to Django's `blank` error code.